spec/services/search/twitter_service_spec.rb

```diff
@@ -46,4 +50,31 @@
       end
     end
   end
+
+  describe '.make_an_author_by' do
+    let(:tweet) { Twitter::Tweet.new(id: '596008529309278208', text: 'a text tweet',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.